### ISSUE TITLE
Babel: publish deprecation notice for babel-preset-react-native

### ISSUE THREAD

**Issue by ricardobeat** [facebook/metro#240]
It is quite hard to follow the trail and figure out that `babel-preset-react-native` has been discontinued:

- the README for [babel-preset-react-native in npm](https://www.npmjs.com/package/babel-preset-react-native) doesn't mention it
- the `repository` field in package.json [points to a 404](https://github.com/facebook/react-native/tree/master/babel-preset/README.md)

The only findable trace of the change is this commit: https://github.com/facebook/react-native/commit/35bd17f1d6a99368ecf6a7f94902bd550827d0d3

It would be helpful if an update to `babel-preset-react-native` would be published with a deprecation warning in `README`, plus a link to the new package and an updated `repository.url`.

(context: we are parsing our codebase using a few custom babel plugins in CI, and `babel-preset-react-native` is the used preset)

**Comment by rafeca** [comment]
Thanks for reporting this! you're right I'll release a new major empty version of the plugin with the deprecation notice

**Comment by rafeca** [comment]
I've published `babel-preset-react-native` v4.0.1, with the same contents of v4.0.0 (since there are some old scripts in React Native that try to install the latest version of the package) and the deprecation message:

https://www.npmjs.com/package/babel-preset-react-native

Closing this issue, thanks again!
